boards/arm/96b_carbon/96b_carbon.dts

```diff
@@ -41,6 +41,14 @@
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
 
+&spi1 {
+	status = "ok";
```

Comment:
If you don't modify IRQ prio here, it will be 0 instead of 5 as set in Kconfig.defconfig

Reply:
I configured default SPI IRQ prio to 5 on SOC level.